src/components/custom/dashboard/shared/chat/RealTimeChatInterface.jsx

```
@@ -0,0 +1,914 @@
+              </>
+            )}
+          </>
+        )}
+      </div>
+
+      {/* Reply form */}
+      {isBlocked && session?.user?.role === "trainer" ? (
+        <div className="border-t border-[#333] p-4 bg-orange-900/20">
+          <div className="flex items-center gap-3 text-orange-300">
+            <svg className="h-5 w-5" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+              <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M18.364 18.364A9 9 0 005.636 5.636m12.728 12.728L5.636 5.636m12.728 12.728L18.364 5.636M5.636 18.364l12.728-12.728" />
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)